site/content/blog/release-0.30.0.md

```diff
@@ -0,0 +1,174 @@
+API tries to resemble those types as much as reasonable (deref coercions etc.), so it should be
+intuitive. Methods in the library that previously took `&Script` (which is now `ScriptBuf`) take the
+unsized `&Script` now.
+
+Additionally, we changed the type accepted by the `push_slice` method to be another unsized newtype:
+`PushBytes` and it's owned counterpart `PushBytesBuf`. These types maintain the invariant of storing
+at most 2^32-1 bytes - the maximum one can push into script. Previously the method would panic if
+you attempted to do it (and it wasn't documented, sorry about that). Now you can either handle it
+explicitly using `TryFrom` or just pass a known-length array (implemented for arrays up to 73
+bytes).
+
+Types that are commonly pushed into script (serialized signatures, public keys...) implement
+`AsRef<PushBytes>` so you can pass those directly as well. You can also implement `AsRef` for your
+types so they can be pushed directly.
+
+## Sighash
+
+We moved around and renamed a bunch of types to do with sighashes. In the `sighash` module, along
+with the `SighashCache` we now have the hopefully clearly named:
+
+- `LegacySighash`
+- `SegwitV0Sighash`
+- `TapSighash`
+- `EcdsaSighashType`
+- `TapSighashType`
+
+Signatures are now in their respective modules (`ecdsa` for legacy and segwit v0):
+
+- `taproot::Signature`
+- `ecdsa::Signature`
+
+## Lock types
+
+There are now two lock times, one for absolute locks (CLTV) and one for relative locks (CSV). We
+export the `absolute` and `relative` modules at the crate root so you can either import them from
+there or `use bitcoin::locktime::{absolute, relative};` if that's clearer. We expect locks to be
+used as `absolute::LockTime`.
+
+## Address changes
+
+Bitcoin addresses for different networks are different, up until this release, when parsing an
+address from a string, a check that the address format matched up to the expected network (e.g. "bc1"
+prefix for Bitcoin mainnet segwit addresses) was available but easy to forget. We've attempted to
+improve the API to make such omissions harder.
+
+Now `Address<V>` includes a generic that is used as a marker for whether the address has been
+checked as valid for a particular network, we have `Address<NetworkChecked>` and
+`Address<NetworkUnchecked>`, defaulting to `NetworkChecked`. Because of the default some uses will
+just keep working but you should be aware that `Address` now means `Address<NetworkChecked>`. The
+string parsing functions return types as expected. See the docs on `Address` for more information.
+
+## Newtypes
+
+This is a non-exhaustive list of newtypes added this release:
+
+- `absolute::LockTime`
+- `absolute::Height`
+- `absolute::Time`
+- `ecdsa::SerializedSignature`
+- `ScriptBuf`
+- `Target`
+- `CompactTarget`
+- `Work`
+
+We added a `U256` type but did not expose it since it is not a general purpose integer type. Rather
```

Comment:
It was [already present](https://docs.rs/bitcoin/0.29.2/bitcoin/util/uint/struct.Uint256.html), we've hid it behind `Work` and `Target`. That's why I suggested it in the section below.

Reply:
oh my bad, I wondered why you wrote `Uint256`, I had totally forgotten that it used to be called that.

Reply:
I think we should just drop this sentence now, since we mention `Uint256` below. We probably don't need to say that it was renamed, since it's private anyway.

Reply:
This sentence is still present.

Reply:
Thanks for flagging the comment, I had indeed missed it.

The `Uint256` type was public before this release so I rekon its worth a mention. Will remove if you feel strongly about it though. Maybe I'm biased because I did the work re-writing that type :)

Reply:
Oh I was only looking at the bottom lines refering to it, will remove the top one. Thanks for you patience.